components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -440,36 +484,38 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
     // Step 4: In-cluster resources set up
     let status = '';
     let getAttempts = 0;
-    const getTimeout = 15000;
+    const getTimeout = 20000;
     do {
       getAttempts++;
+      await wait(getTimeout);
```

Comment:
Why move the wait up here? We should only wait after each call fails right? I think that's what the old code does.

Reply:
We want to always wait:
even call succeeded, if the returned status is "RUNNING", we will still wait for 20s.
So in total we wait up to 20*30 (10 min), till deployment manager successfully set up every thing (usually finish within 5 min).